src/sql/workbench/browser/modal/optionsDialog.ts

```diff
@@ -201,6 +201,7 @@ export class OptionsDialog extends Modal {
 
 			let serviceOptions: azdata.ServiceOption[] = categoryMap[category];
 			let bodyContainer = $('table.optionsDialog-table');
+			bodyContainer.setAttribute('role', 'list');
```

Comment:
this should not be a list, I'd rather set this to 'presentation', it is purely used to do layout.

Reply:
It seems like a list of sorts to me - what makes you say this isn't a list?

Reply:
https://www.w3.org/WAI/PF/HTML/wiki/RoleAttribute

![image](https://user-images.githubusercontent.com/13777222/128751500-f1dc6070-7adc-4c37-b3cc-b7ee144f1376.png)